### PR TITLE
Remove the montserrat font from PCUI

### PR DESCRIPTION
**Pull request by ellthompson**
This PR changes the PCUI build to use the default browser font. Users can build PCUI with their own desired font if necessary.

### CODE REVIEW

File: src/scss/fonts.scss [hunk 1]
```diff
@@ -11,72 +11,17 @@
...
-    font-style: normal;
-}
-
-.font-light {
-    font-family: 'Montserrat Light', 'Helvetica Neue', Arial, Helvetica, sans-serif;
-    font-weight: 200;
-    font-style: normal;
-}
-
 .font-regular {
```
Comment (ellthompson) [inline src/scss/fonts.scss:20]: I've left the font-regular rule in as this is extended in multiple other scss files.

Comment (vkalpias) [reply src/scss/fonts.scss:20]: Can we keep the classes but just remove the font-families? e.g. font-bold is used too

Comment (ellthompson) [reply src/scss/fonts.scss:20]: Sure

**Review by vkalpias — APPROVED** [review]

File: src/scss/pcui-storybook.scss [hunk 2]
```diff
@@ -3,6 +3,10 @@
 @import 'pcui-common.scss';
 @import 'pcui-hidden.scss'; // this should be last
 
+.font-regular, .font-bold, .font-thin, .font-light {
+    font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
```
Comment (vkalpias) [inline src/scss/pcui-storybook.scss:7]: If we're adding those here maybe we can include the Montserrat font as well in this bit?

**Review by vkalpias — APPROVED** [review]

File: src/scss/pcui-storybook.scss (hunk 2 above)
Comment (ellthompson) [reply src/scss/pcui-storybook.scss:7]: Yeah sure, i'll add those fonts in here.

**Pull request merged**